changelog.md

```diff
@@ -0,0 +1,36 @@
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added 
+
+### Fixed
+
```

Comment:
This is probably not supposed to be here

Reply:
Nop, that was on purpose to serve as guideline for the following PRs

Reply:
As in people should add entries to that "Added" and "Changed" sections under `Unreleased` and once we do make a release we'll change the title to whatever release is suitable and add a new `Unreleased`